### DISCUSSION TITLE
New  "action" color class request

### DISCUSSION THREAD

**Discussion by ScotsScripts** [twbs/bootstrap#35494] [Ideas]
Wondering if devs would consider adding a new color class, "action", to the standard bootstrap color classes?

So much dev work is done in ecommerce that I think having a dedicated "action" color is necessary, esp since bootstrap names it's colors by roles (which is super cool.) 

btn-action, text-action, bg-action, etc etc... 

Stock btn-warning yellow is a good action color, but doesn't make sense to use "warning" classes on action items.

**Comment by florianlacreuse** [comment]
It seems unlikely to me.

Good news is that you can add colors to the theme colors map: https://getbootstrap.com/docs/5.1/customize/sass/#add-to-map

```scss
$custom-colors: (
  "action": #7952b3
);

$theme-colors: map-merge($theme-colors, $custom-colors);
```
